cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -55,6 +55,10 @@ package_upgrade: false
 repo_upgrade: none
 
 datasource_list: [ Ec2, None ]
+
+${DisableSudoAccessForDefaultUserConfig}
```

Comment:
This config references the `default_user` which is not defined within cloud-init config (it is in the CloudInit copnfig for Batch).
I suppose the default user is defined implicitly by CloudInit, but may you please confirm?

Reply:
Yes it is. The below section that you saw in cloud-init config for Batch is the same for every node or ec2 instance.
```

system_info:
  default_user:
    sudo:
    - ALL=(ALL) !ALL
```

The `${DisableSudoAccessForDefaultUserConfig}` replaces the above section I had harcoded for Batch.

Its just that I hardcoded it in Batch as I wanted to avoid using the if else conditions in Batch. But sake of simplicity I will add that.